c2corg_ui/static/js/card.js

```diff
@@ -215,9 +215,7 @@ app.CardController.prototype.getGlobalRatings = function() {
       ratings['biking_rating'] = this.slashSeparatedRating_(doc.mtb_down_rating, doc.hiking_mtb_exposition);
     }
   }.bind(this));
-
-  ratings = Object.keys(ratings)[0] ? ratings : null;
-  return ratings;
+  return ratings[Object.keys(ratings)[0]] ? ratings : null;
```

Comment:
Why not simply
```
return Object.keys(ratings).length ? ratings : null;
```
?

Btw, is it really a problem to return an empty object ``{}``? Aren't the card partials able to handle it?